mountainduck/preferences.md

```diff
@@ -137,7 +139,9 @@ The connection profiles for [default protocols](../protocols/index.md) are alway
 
 ## Login Item
 
-Reconnect after restarting the computer. If you choose *Enable Login Item* and *Save Workspace* in *Preferences → General* and do not manually eject the volume prior to reboot it will reconnect after login.
+Reconnect after restarting the computer. 
+
+If you enable *Enable Login Item* and [*Save Workspace*](#save-workspace) in *Preferences → General* and do not manually eject the volume prior to reboot it will reconnect after login.
```

Comment:
```suggestion
If you enable *Enable Login Item* and [*Save Workspace*](#save-workspace) in *Preferences → General* and do not manually eject the volume prior to quitting Mountain Duck, it will reconnect after login.
```